user/customizing-the-build.md

```diff
@@ -261,17 +261,52 @@ git:
   submodules: false
 ```
 
-## Git LFS Skip Smudge
+## Git LFS
 
-You can disable the download of LFS objects when cloning ([`git lfs smudge
---skip`](https://github.com/git-lfs/git-lfs/blob/master/docs/man/git-lfs-smudge.1.ronn))
-by setting the following in `.travis.yml`:
+
+### Authentication
+
+As a general rule, we recommend using authenticated requests to avoid being rate-limited by GitHub:
```

Comment:
Probably not worth opening another PR for now this is merged, but

> As a general rule, 

adds very little. I'd probably reword to something like

> We recommend using a read-only GitHub OAuth token to authenticate when using git LFS, this is required when connecting to private repositories, and prevents rate-limiting when connecting to open source repositories:

And I'd be tempted to add a single sentence description / link to what LFS is before the authentication section.

:-)